plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -52,8 +52,8 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 	return
 }
 
-func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
```

Comment:
remove getTolerationListFromPod and just have getAllTolerationPreferNoSchedule, it seems unnecessary/confusing to wrap a specific function for PreferNoSchedule.